packages/backend/src/machine-utils.ts

```diff
@@ -80,7 +80,16 @@ export async function isPodmanV5Machine() {
     const machineInfo = await getMachineInfo();
 
     const ver = machineInfo.Version.Version;
-    return satisfies(ver, '>=5.0.0');
+    // Attempt to parse the version, handling undefined if it fails
+    const coercedVersion = coerce(ver);
+    if (!coercedVersion) {
+      // Handle the case where the version could not be coerced successfully
+      console.error('Unable to parse Podman machine version:', ver);
+      return false;
```

Comment:
ah, so I added this in case the version for some reason is blank in the configuration / malformed configuration, which one probably indicate a bigger problem and that's why I return false, similar to the other below code.